vendor/assets/javascripts/dvl/vendor/bootstrap-datepicker.js

```diff
@@ -0,0 +1,1681 @@
+    }
+    else {
+      this.picker.addClass('datepicker-dropdown dropdown-menu');
+    }
+
+    if (this.o.rtl){
+      this.picker.addClass('datepicker-rtl');
+    }
+
+    this.viewMode = this.o.startView;
+
+    if (this.o.calendarWeeks)
+      this.picker.find('tfoot th.today, tfoot th.clear')
+            .attr('colspan', function(i, val){
+              return parseInt(val) + 1;
+            });
+
+    this._allow_update = false;
+
+    this.setStartDate(this._o.startDate);
+    this.setEndDate(this._o.endDate);
+    this.setDaysOfWeekDisabled(this.o.daysOfWeekDisabled);
+
+    this.fillDow();
+    this.fillMonths();
+
+    this._allow_update = true;
+
+    this.update();
+    this.showMode();
+
+    if (this.isInline){
+      this.show();
+    }
+  };
+
+  Datepicker.prototype = {
+    constructor: Datepicker,
+
+    _process_options: function(opts){
+      // Store raw options for reference
+      this._o = $.extend({}, this._o, opts);
+      // Processed options
+      var o = this.o = $.extend({}, this._o);
+
+      // Check if "de-DE" style date is available, if not language should
+      // fallback to 2 letter code eg "de"
+      var lang = o.language;
+      if (!dates[lang]){
+        lang = lang.split('-')[0];
+        if (!dates[lang])
+          lang = defaults.language;
+      }
+      o.language = lang;
+
+      switch (o.startView){
+        case 2:
+        case 'decade':
+          o.startView = 2;
+          break;
+        case 1:
+        case 'year':
+          o.startView = 1;
+          break;
+        default:
+          o.startView = 0;
+      }
+
+      switch (o.minViewMode){
+        case 1:
+        case 'months':
+          o.minViewMode = 1;
+          break;
+        case 2:
+        case 'years':
+          o.minViewMode = 2;
+          break;
+        default:
+          o.minViewMode = 0;
+      }
+
+      o.startView = Math.max(o.startView, o.minViewMode);
+
+      // true, false, or Number > 0
+      if (o.multidate !== true){
+        o.multidate = Number(o.multidate) || false;
+        if (o.multidate !== false)
+          o.multidate = Math.max(0, o.multidate);
+        else
+          o.multidate = 1;
+      }
+      o.multidateSeparator = String(o.multidateSeparator);
+
+      o.weekStart %= 7;
+      o.weekEnd = ((o.weekStart + 6) % 7);
+
+      var format = DPGlobal.parseFormat(o.format);
+      if (o.startDate !== -Infinity){
+        if (!!o.startDate){
+          if (o.startDate instanceof Date)
+            o.startDate = this._local_to_utc(this._zero_time(o.startDate));
+          else
+            o.startDate = DPGlobal.parseDate(o.startDate, format, o.language);
+        }
+        else {
+          o.startDate = -Infinity;
+        }
+      }
+      if (o.endDate !== Infinity){
+        if (!!o.endDate){
+          if (o.endDate instanceof Date)
+            o.endDate = this._local_to_utc(this._zero_time(o.endDate));
```

Comment:
Expected '{' and instead saw 'o'.<br>Identifier '_local_to_utc' is not in camel case.<br>Identifier '_zero_time' is not in camel case.